### ISSUE TITLE
Remove endpoints informer

### ISSUE THREAD

**Issue by josephburnett** [pivotal/skenario#68]
Without Knative buffering and routing, we don't need to inject an endpoints informer into replicas.

**Comment by jchesterpivotal** [comment]
Yep. Compare the defunct `060-upgrade` branch, where I was [able to successfully remove the endpoints informer](https://github.com/pivotal/skenario/commit/96d9ccf12e9798931183421fe4866968ba18a4bb).

Getting rid of that dependency was a big win IMO.
